packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/SimulationEventService.java

```diff
@@ -95,21 +106,23 @@ private void onScimlSendToUserEvent(final Message message, final Channel channel
      * @throws IOException if there was an error sending the message
      */
     @RabbitListener(
-            queues = {PYCIEMSS_QUEUE},
+            queues = "${terarium.simulation-status}${terarium.queue.suffix:${terarium.userqueue.suffix}}",
             concurrency = "1")
     private void onPyciemssSendToUserEvent(final Message message, final Channel channel) throws IOException {
-        // TODO this should mirror the implementation of onScimlSendToUserEvent
+
+			/*
+				//TODO implement PyciemssStatusUpdate
```

Comment:
Is there an issue we can link to this todo?

Reply:
There is no issue... logging one would be good but I don't know who or what it would be or what the status/holdup is. @mwdchang ?

Reply:
I will create an issue if none exist - the hold up is ciemss is doing a big refactor over their codebase and we are not expecting to have a usable intermediate result in the next short little while.

Reply:
In theory once they're done their refactor then the only backend work will be to implement the `PyciemssStatusUpdate` model. Everything else should automagically work.